sourced/ml/algorithms/uast_struct_to_bag.py

```diff
@@ -87,103 +100,77 @@ def __init__(self, p_explore_neighborhood, q_leave_neighborhood, n_walks, n_step
         self.q_leave_neighborhood = q_leave_neighborhood
         self.n_walks = n_walks
         self.n_steps = n_steps
-        self.type2ind = type2ind if type2ind is not None else FakeVocabulary()
+        self.node2index = node2index if node2index is not None else Node2InternalType()
+        if seed is not None:
```

Comment:
Move seed here from `UastRandomWalk2Bag ` class because here all random parts.

Note, that If earlier it was inside `uast_to_bag` function, now it is in `__init__`. And I think it is better.